Client/Frontend/Settings/Brave News/BraveNewsSettingsViewController.swift

```diff
@@ -74,7 +79,13 @@ class BraveNewsSettingsViewController: TableViewController {
                 rows: [
                     .boolRow(
                         title: Strings.BraveNews.isEnabledToggleLabel,
-                        option: Preferences.BraveNews.isEnabled
+                        option: Preferences.BraveNews.isEnabled,
+                        onValueChange: { value in
+                            Preferences.BraveNews.isEnabled.value = value
+                            if value {
+                                self.rewards?.ads.initialize { _ in }
```

Comment:
Should self be unowned like in other places?

Reply:
Good call, it should :)